cpp/src/arrow/sparse_tensor/coo_converter.cc

```diff
@@ -0,0 +1,192 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "arrow/sparse_tensor/converter.h"
```

Comment:
Can you run IWYU on this file?

Reply:
I added a commit to fix for IWYU, but the following reports are remaining.  I guess these reports are false positive.

```
include-what-you-use 0.13 (git:06e88ef) based on clang version 9.0.0-2~ubuntu18.04.2 (tags/RELEASE_900/final)
/home/mrkn/src/github.com/apache/arrow/cpp/src/arrow/tensor/csf_converter.cc should add these lines:
namespace arrow { template <typename T, typename Enable = void> class TypedBufferBuilder; }

/home/mrkn/src/github.com/apache/arrow/cpp/src/arrow/tensor/csf_converter.cc should remove these lines:
- #include "arrow/buffer_builder.h"  // lines 27-27
```

Reply:
Seems so